src/main/java/com/odysseusinc/arachne/portal/api/v1/controller/BaseAchillesController.java

```diff
@@ -98,10 +99,13 @@ public BaseAchillesController(BaseDataSourceRepository<DS> dataSourceRepository,
     public void receiveStats(
```

Comment:
Why it should start working for regular user?

@PreAuthorize("#ds.dataNode == authentication.principal")

Reply:
see https://github.com/OHDSI/ArachneCentralAPI/pull/261 pr